Misc/NEWS.d/next/Tests/2023-02-21-23-23-26.bpo-95672.-ogWWJ.rst

```diff
@@ -0,0 +1,2 @@
+Fix the typos ("SkitTest") in test_fcntl_f_pipesize and test_pipesizes.
```

Comment:
@hyeongyun0916 We can skip the NEWS.d since the unit test does not affect to end-user usage :)

Reply:
Okay I'll delete it